src/stores/ErrorStore.js

```diff
@@ -0,0 +1,25 @@
+// @flow
```

Comment:
I'm not sure about this store at all...may be there is better solution, for handling errors.

Reply:
This is fine for now. We can look into improving once the app grows larger